recipes/spectate/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "spectate" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d4d985589d755b4bf118696eb794bc35c4131809be812eecd95fd9938809efbf
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
I believe it's good practice to add a `--no-deps` here to prevent pip from installing dependencies from pypy :)

Reply:
k thanks! =)